src/browser.ts

```diff
@@ -10,15 +10,23 @@ import { mkdtemp } from 'fs/promises'
 import path from 'path'
 import os from 'os'
 import { appSettings } from './main'
+import { exec, spawn } from 'child_process'
+import { getPlatform } from './utils/electron'
+import log from 'electron-log/main'
 
 const createUserDataDir = async () => {
   return mkdtemp(path.join(os.tmpdir(), 'k6-studio-'))
 }
 
-function getBrowserPath() {
+async function getBrowserPath() {
   const { recorder } = appSettings
 
   if (recorder.detectBrowserPath) {
+    if (getPlatform() === 'linux' && process.arch === 'arm64') {
+      // Chrome is not available for arm64, use Chromium instead
+      return await getChromiumPath()
+    }
```

Comment:
Chrome isn't available on Linux arm64 so we're defaulting to Chromium instead.

Reply:
We could actually attempt both in all platforms, starting with Chrome and having Chromium as a fallback 🤔

Reply:
I think this is the way to go: If Chromium is supported, one doesn't have to have Chrome installed on Windows/Mac to use Recorder.